js/script.js

```diff
@@ -0,0 +1,224 @@
+// GET ALL BOXES
+var $boxes = $('.box')
+var $wrapper = $('.wrapper')
+
+//allow movement?
+var clicker=true
+
+function initGame() {
+  var snd = new Audio("./audio/scary.mp3"); // buffers automatically when created
+  snd.play();
+
+  // SET THE MAP
+  map.forEach(function(spotY,indexY) {
+    spotY.forEach(function(spot, indexX){
+    switch (spot) {
```

Comment:
Good `'switch` approach with many cases. :)

And you may do this with 2-D Array and `in` statement for replacing each switch statement.

e.g.
`var arr = [[1,5,6,9,11,12,14], [ ...... ], [......], .........]`
`if( spot in arr[0] ) $wrapper.eq(indexY).find('.box').eq(indexX).addClass('top')`